plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqCodeTest/CodeWhispererUTGChatManager.kt

```diff
@@ -517,8 +554,9 @@ class CodeWhispererUTGChatManager(val project: Project, private val cs: Coroutin
                     jobGroup = session.testGenerationJobGroupName,
                     jobId = session.testGenerationJob,
                     result = if (e.message == message("testgen.message.cancelled")) MetricResult.Cancelled else MetricResult.Failed,
-                    reason = e.javaClass.name,
-                    reasonDesc = e.message,
+                    reason = (e as CodeTestException).code ?: "DefaultError",
```

Comment:
why don't we just let it be null?

Reply:
It can be but placing this under the bucket of DefaultError to be consistent in Kibana.